lib/mayaUsd/resources/ae/usdschemabase/ae_template.py

```diff
@@ -0,0 +1,124 @@
+import fnmatch
+import ufe
+import mayaUsd.ufe as mu
```

Comment:
mu?

Reply:
Does that matter? Isn't this just a name to use instead of "mayaUsd.ufe.xxx"

Reply:
You are absolutely correct in saying that there is no problem on executing the above code and getting correct results.  Just wondering about the very terse name...  m can be Maya or maya-usd, u can be USD or UFE...  Kinda hard to find a more descriptive name that's shorter than mayaUsd.ufe, I guess...  You can leave it as is if you like.

Reply:
Your right. I only used it once, so I've just removed that